examples/ckks/bootstrapping/main.go

```diff
@@ -95,7 +120,7 @@ func main() {
 	// To equalize the scale, the function evaluator.SetScale(ciphertext, parameters.DefaultScale()) can be used at the expense of one level.
```

Comment:
Please see my comment on the `Bootstrapp()` function. The condition documented in this comment should be checked in code.